sdcm/tester.py

```diff
@@ -905,10 +941,30 @@ def run_fullscan_thread(self, ks_cf='random', interval=1, duration=None):
         """
         duration = self.get_duration(duration)
         interval = interval * 60
-        self.loaders.run_fullscan_thread(ks_cf,
-                                         db_nodes=self.db_cluster.nodes,
-                                         interval=interval,
-                                         duration=duration)
+
+        @log_run_info('Fullscan thread')
+        def run_in_thread():
+            start = current = time.time()
+            while current - start < duration:
+                loader_node = random.choice(self.loaders.nodes)
+                db_node = random.choice(self.db_cluster.nodes)
+                if db_node.termination_event.isSet():
+                    break
+                if hasattr(db_node, 'remoter') and db_node.remoter and not db_node.remoter.is_up():
+                    self.log.warning("Chosen node is down. Choose new one")
+                    continue
+
+                self.run_fullscan(ks_cf, loader_node, db_node)
+                time.sleep(interval)
+                current = time.time()
+
+        th = threading.Thread(target=run_in_thread)
+        th.daemon = True
+        th.start()
+
+    def kill_fullscan_thread(self):
```

Comment:
don't think we need this, `termination_event` is general for test end, remove this and a call  this method

Reply:
Fixed